bim2sim/project.py

```diff
@@ -491,7 +495,8 @@ def run(self, interactive=False, cleanup=True):
 
         if not self._user_logger_set:
             logger.info("Set user logger to default Stream. "
-                        "Call project.set_user_logger_stream() prior to project.run() to change this.")
+                        "Call project.set_user_logger_stream() prior to "
```

Comment:
set_user_logger_stream() doesn't exist. 

How do I change the logging level when running an example?

Reply:
> set_user_logger_stream() doesn't exist.

I fixed this with latest commit.

> How do I change the logging level when running an example?

If you want to put the user log to another place so you don't see everything in console you could do the following:
### change user log handler:
* This is described in the [documentation of the logging concept ](https://ebc.pages.rwth-aachen.de/EBC_all/github_ci/bim2sim/development/docs/concepts/logging.html)
* E.g. you could add these statements before running `project.run()`

```
    from logging import FileHandler

    my_handler = FileHandler("my_log_file.log")

    project.set_user_logging_handler(my_handler)
```
* Then you won't have any log messages in the console from the user logging as they will be writen to the my_lof_file.log file in your CWD.

### If you just want to change the threshold level from which on you get logs you can do the following:
```
    general_logger = logging.getLogger('bim2sim')

    general_logger.setLevel(logging.WARNING)
```
This way you will only receiver Warnings and Errors and Critials, see [logging module docs](https://docs.python.org/3/library/logging.html#levels) as well